### ISSUE TITLE
False positive for `new Error().stack`

### ISSUE THREAD

**Issue by sindresorhus** [dustinspecker/eslint-plugin-no-use-extend-native#7]
```
$ eslint --version
1.7.3
```

```
$ echo 'new Error().stack' | eslint --stdin --plugin='no-use-extend-native'

<text>
  1:1  error  Avoid using extended native objects  no-use-extend-native/no-use-extend-native

✖ 1 problem (1 error, 0 warnings)
```

Noticed it in https://github.com/sindresorhus/ava/blob/85bfb22112a6bdfb825aeb54c7186d162d2c0d23/lib/test.js#L82
